utils/build-script

```diff
@@ -353,6 +353,21 @@ class BuildScriptInvocation(object):
             args.skip_test_tvos = True
             args.skip_test_watchos = True
 
+        # --build-swift-stdlib is an alias for --build-swift-dynamic-stdlib=1
+        # and --build-swift-static-stdlib=0, which takes precedence over those
+        # options.
+        if args.build_swift_stdlib:
+            args.build_swift_dynamic_stdlib = True
+            args.build_swift_static_stdlib = False
```

Comment:
Is there a good way to determine whether `--build-swift-stdlib` is specified on the command line? Right now it seems like the following are indistinguishable:

```
$ utils/build-script
$ utils/build-script --build-swift-stdlib=0
```

Is there a good way to distinguish these two using the existing `swift_build_support` code? Or should I make some modifications there, or just inspect `argv`?

Thanks for all your help!